media/linux/pds-sqlite3-queries/create-training-rosters.py

```diff
@@ -316,17 +270,18 @@ def main():
                     'api_name'    : 'drive',
                     'api_version' : 'v3', },
     }
-    services = GoogleAuth.service_oauth_login(apis,
-                                              app_json=args.app_id,
-                                              user_json=args.user_credentials,
-                                              log=log)
-    google = services['drive']
+    google = ''
+    if not args.dry_run:
+        services = GoogleAuth.service_oauth_login(apis,
+                                                app_json=args.app_id,
+                                                user_json=args.user_credentials,
+                                                log=log)
+        google = services['drive']
 
     for training in trainings:
-        create_roster(pds_members=pds_members,
-                      training=training,
-                      google=google,
-                      log=log)
+        trainingdata = find_training(pds_members, training)
```

Comment:
```suggestion
        training_data = find_training(pds_members, training)
```

Maybe name the function `pds_find_training()`, to emphasize that you're searching PDS data to find the training.

Also, use `training_data`.  If you use snake case, be consistent.  😄